python/packages/autogen-core/src/autogen_core/models/_model_client.py

```diff
@@ -36,6 +37,7 @@ async def create(
         cancellation_token: Optional[CancellationToken] = None,
     ) -> CreateResult: ...
 
+    @abstractmethod
```

Comment:
Remember we need to let's add the '*' before the keyword arguments

Reply:
Yeah but in a different PR right?

Reply:
Yes